content/src/service/ServiceImpl.ts

```diff
@@ -288,7 +287,17 @@ export class ServiceImpl implements MetaverseContentService {
               deploymentId,
               entity
             )
-            affectedPointers = Array.from(pointersFromEntity.keys())
+
+            // Update pointers to entityId cache
+            for (const pointerChange of pointersFromEntity) {
+              if (pointerChange[1].after === DELTA_POINTER_RESULT.CLEARED) {
```

Comment:
Please add a comment here explaining that this means that the entity has been overwritten